### PR TITLE
KOGITO-4233: [DMN Designer] IntelliSense service

### PR DESCRIPTION
**Pull request by karreiro**
**JIRA**: [KOGITO-4233](https://issues.redhat.com/browse/KOGITO-4233): [DMN Designer] IntelliSense service

**Part of an ensemble**:
- https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1611
- https://github.com/kiegroup/drools/pull/3435
- https://github.com/kiegroup/appformer/pull/1115
- https://github.com/kiegroup/kie-wb-common/pull/3590
- https://github.com/kiegroup/drools-wb/pull/1471
- https://github.com/kiegroup/kie-wb-distributions/pull/1091

**Artifacts**:
- **Business Central**: [WAR file](https://www.dropbox.com/s/fu3ra76qhptzprr/KOGITO-4233-bc.war?dl=0)
- **VS Code**: [plugin](https://www.dropbox.com/s/jaxmlnilmexk1zw/KOGITO-4233-plugin.vsix?dl=0)

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by Rikkola** [comment]
Jenkins please retest this.

File: uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/monaco/jsinterop/ITextModel.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
```
Comment (Rikkola) [inline uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/java/org/uberfire/client/views/pfly/monaco/jsinterop/ITextModel.java:2]: Not sure if we need to change this to 2021.

**Review by Rikkola — APPROVED** [review]
No tests, but this will be tested by the editors themselves. ( Says me who wrote it )

**Comment by karreiro** [comment]
Jenkins run cdb.

**Comment by karreiro** [comment]
Jenkins run cdb.

**Comment by jesuino** [comment]
jenkins execute cdb

**Review by danielzhe — APPROVED** [review]

**Review by jomarko — APPROVED** [review]

**Pull request merged**